xcm/src/v3/mod.rs

```diff
@@ -78,6 +79,24 @@ impl<Call> Xcm<Call> {
 		self.0.len()
 	}
 
+	/// Return a reference to the inner value.
+	pub fn inner(&self) -> &Vec<Instruction<Call>> { &self.0 }
+
+	/// Return a mutable reference to the inner value.
+	pub fn inner_mut(&mut self) -> &mut Vec<Instruction<Call>> { &mut self.0 }
+
+	/// Consume and return the inner value.
+	pub fn into_inner(self) -> Vec<Instruction<Call>> { self.0 }
+
+	/// Return an iterator over references to the items.
+	pub fn iter(&self) -> impl Iterator<Item=&Instruction<Call>> { self.0.iter() }
+
+	/// Return an iterator over mutable references to the items.
+	pub fn iter_mut(&mut self) -> impl Iterator<Item=&mut Instruction<Call>> { self.0.iter_mut() }
+
+	/// Consume and return an iterator over the items.
+	pub fn into_iter(self) -> impl Iterator<Item=Instruction<Call>> { self.0.into_iter() }
```

Comment:
We can just implement `IntoIterator` for the `Xcm` type instead of this being an inherent method. It will give us a nicer `for` loop syntax such as `for msg in xcm` without the need to call `into_iter()`.

Reply:
Sure - feel free to make the change.